src/index.d.ts

```diff
@@ -39,6 +40,13 @@ type QueryHookOptions<TVariables> = Omit<QueryOptions<TVariables>, 'query'> & {
   suspend?: boolean;
 };
 
+type SubscribeToMoreHookOptions = Omit<
+  SubscribeToMoreOptions<TData, TVariables, TSubscriptionData>,
+  'updateQuery'
+> & {
+  updateQuery: UpdateQueryFn<TData, TVariables, TSubscriptionData>;
```

Comment:
updateQuery is market as optional in the source, however, without it, subscribeToMore has no side effects, [see code](https://github.com/apollographql/apollo-client/blob/911ca2db631adc38745b8098dd802e0cdfe8dbbb/packages/apollo-client/src/core/ObservableQuery.ts#L367)